tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.hh

```diff
@@ -265,6 +265,22 @@ protected:
     // Opens output files as stringstreams
     void openOutputFiles(EnergyPlusData &state);
 
+    // A worker function that keeps trailing spaces in multiline raw string literals
+    void replace_pipes_with_spaces(std::string &stringLiteral)
+    {
+        // C++ now allows nice raw multiline string literals
+        // These are very useful for unit tests that compare against a long chunk of text
+        // For example, if you have an EIO output to compare, you can just paste it right into a variable and compare
+        // However, if the raw literal has trailing spaces on any line, editors tend to remove them
+        // I tried disabling clang-format for sections of code, I tried using an .editorconfig file, and still
+        // the editor would remove them on saving the file.  And for my editor, I _can_ disable that, but only for all
+        // files, which is not preferred.  We do want trailing spaces stripped, just not inside raw string literals.
+        // Anyway, the solution I came up with is a small worker function here.  In the string literal, just put in
+        // pipe characters ( "|" ) where the trailing spaces would be.  Then pass your string literal into this
+        // function, and it will programmatically replace them with strings.  You can then do asserts as needed.
+        std::replace(stringLiteral.begin(), stringLiteral.end(), '|', ' '); // replace the trailing || with spaces
+    }
```

Comment:
This could be made a bit more general purpose, but I think the simpler the better in this case.  This should help clean up the long chunks of delimited strings and allow them to be raw string literals throughout the unit test suite.